shared/src/test/diff/mlscript/BadMethods.mls

```diff
@@ -9,9 +9,14 @@ class Bar[A]: Foo[A]
 //│ ║  l.7: 	    rec method Map f = f Map (fun x -> Map)
 //│ ║       	               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 //│ ╟── expression of type `B` is not a function
+//│ ║  l.6: 	    method Map[B]: B -> A
```

Comment:
It looks like this is an origin-prov being printed. It should not.

Reply:
Yes, but otherwise nothing is printed because the list of provenance is empty except for the origin prov. I figured origin prov should not be printed when we had more precise provenances, but in this case, they might be of interest?

Reply:
Hmm sure, why not. But then it shouldn't be printed in the note below, because it's redundant. Anyway, this part of the code will change soon.